src/lib.rs

```diff
@@ -22,9 +22,6 @@
 //! ```
 //!
 //! [zstd]: https://github.com/facebook/zstd
-#![deny(missing_docs)]
```

Comment:
Let's not remove that one :)

Reply:
Sorry, I thought it belongs to libc crate 😄 I will fix it

Reply:
Fixed